packages/calculator-bn.js/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@dinero.js/calculator-bn.js",
+  "version": "1.0.0-alpha.0",
```

Comment:
```suggestion
  "version": "2.0.0-alpha.12",
```

The version should match the project.

Reply:
@johnhooks happy to change it, but could u clarify why? 🙏

Reply:
Because if its part of Dinero it should follow the same SemVer.

Reply:
sorry, i don't understand the logic, maybe i'm missing something. Do all the packages here get released together? For example, if calculator-bigint gets a major version bump, does calculator-bn.js also have to get a major version bump even though nothing changed?

Reply:
Yes everything is release together.